src/components/LeftSidebar/ConversationsList/Conversation.vue

```diff
@@ -302,6 +302,14 @@ export default {
 			}
 		},
 	},
+
+	mounted() {
+		const titleSpan = document.getElementById(`conversation_${this.item.token}`)?.querySelector('.line-one__title')
+		if (!titleSpan) return
+
+		if (titleSpan.offsetWidth < titleSpan.scrollWidth) titleSpan.setAttribute('title', this.item.displayName)
```

Comment:
In Vue we should not directly mutate DOM elements that are fully controlled by Vue's Virtual DOM rendering. DOM should be defined with the component's template. Direct mutating is a very not Vue way. Vue can recreate this element on rendering, then the title will be lost.

But `NcListItem` has no features for setting a native title attribute...

I'd add a `// TODO: ` comment here to fix it in the future when/if NcListItem will support setting the native title attribute or displaying it for long titles.

Related issue: https://github.com/nextcloud/nextcloud-vue/issues/3687